client/js/components/procedure/DpSubmitterList.vue

```diff
@@ -162,7 +180,9 @@ export default {
         isCitizen,
         memo,
         submitName,
-        submitterEmailAddress
+        submitterEmailAddress,
+        initialOrganisationHouseNumber,
+        initialOrganisationStreet
```

Comment:
We could rename the variables while destructuring for better readability like:

```suggestion
        submitterEmailAddress: email,
        initialOrganisationHouseNumber: houseNumber,
        initialOrganisationStreet: street
```

Also, alphabetical sorting.

Reply:
done [here](https://github.com/demos-europe/demosplan-core/pull/672/commits/40d83afd9e1edf4cc43d774edcae6d385aff5134)